### ISSUE TITLE
Tracker doesn't consider new models with pk as changed

### ISSUE THREAD

**Issue by ondrowan** [jazzband/django-model-utils#122]
I don't know if this works as intended, but if I create new model and set pk, `changed()` (and `has_changed()`) returns empty dict.

``` python
>>> model = Model(name='Test', alias='test')
>>> model.tracker.changed()
{'name': None, 'alias': None}

>>> model = Model(name='Test', alias='test', pk=1)
>>> model.tracker.changed()
{}
```

**Comment by webtweakers** [comment]
This seems to be a problem when using a custom primary key that also functions as data and is set before calling the save method. The code @ondrowan refers to fails then.
I believe it's possible to use Model._state.added instead of Model.pk to check for a new instance.

This issue had been reported almost 2 years ago. Has there been no fix for this in the mean time?

**Comment by carljm** [comment]
No, there's been no fix that I'm aware of. I don't personally use FieldTracker, and am feeling inclined to spin it off into its own project. In the meantime, though, I'd review/merge a PR to fix this, if one appeared.
